src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/wrapper/PKCS11Constants.java

```diff
@@ -784,23 +911,80 @@
+    public static final long  CKM_NULL                       = 0x0000400bL;
+    public static final long  CKM_BLAKE2B_160                = 0x0000400cL;
+    public static final long  CKM_BLAKE2B_160_HMAC           = 0x0000400dL;
+    public static final long  CKM_BLAKE2B_160_HMAC_GENERAL   = 0x0000400eL;
+    public static final long  CKM_BLAKE2B_160_KEY_DERIVE     = 0x0000400fL;
+    public static final long  CKM_BLAKE2B_160_KEY_GEN        = 0x00004010L;
+    public static final long  CKM_BLAKE2B_256                = 0x00004011L;
+    public static final long  CKM_BLAKE2B_256_HMAC           = 0x00004012L;
+    public static final long  CKM_BLAKE2B_256_HMAC_GENERAL   = 0x00004013L;
+    public static final long  CKM_BLAKE2B_256_KEY_DERIVE     = 0x00004014L;
+    public static final long  CKM_BLAKE2B_256_KEY_GEN        = 0x00004015L;
+    public static final long  CKM_BLAKE2B_384                = 0x00004016L;
+    public static final long  CKM_BLAKE2B_384_HMAC           = 0x00004017L;
+    public static final long  CKM_BLAKE2B_384_HMAC_GENERAL   = 0x00004018L;
+    public static final long  CKM_BLAKE2B_384_KEY_DERIVE     = 0x00004019L;
+    public static final long  CKM_BLAKE2B_384_KEY_GEN        = 0x0000401aL;
+    public static final long  CKM_BLAKE2B_512                = 0x0000401bL;
+    public static final long  CKM_BLAKE2B_512_HMAC           = 0x0000401cL;
+    public static final long  CKM_BLAKE2B_512_HMAC_GENERAL   = 0x0000401dL;
+    public static final long  CKM_BLAKE2B_512_KEY_DERIVE     = 0x0000401eL;
+    public static final long  CKM_BLAKE2B_512_KEY_GEN        = 0x0000401fL;
+    public static final long  CKM_SALSA20                    = 0x00004020L;
+    public static final long  CKM_CHACHA20_POLY1305          = 0x00004021L;
+    public static final long  CKM_SALSA20_POLY1305           = 0x00004022L;
+    public static final long  CKM_X3DH_INITIALIZE            = 0x00004023L;
+    public static final long  CKM_X3DH_RESPOND               = 0x00004024L;
+    public static final long  CKM_X2RATCHET_INITIALIZE       = 0x00004025L;
+    public static final long  CKM_X2RATCHET_RESPOND          = 0x00004026L;
+    public static final long  CKM_X2RATCHET_ENCRYPT          = 0x00004027L;
+    public static final long  CKM_X2RATCHET_DECRYPT          = 0x00004028L;
+    public static final long  CKM_XEDDSA                     = 0x00004029L;
+    public static final long  CKM_HKDF_DERIVE                = 0x0000402aL;
+    public static final long  CKM_HKDF_DATA                  = 0x0000402bL;
+    public static final long  CKM_HKDF_KEY_GEN               = 0x0000402cL;
+    public static final long  CKM_SALSA20_KEY_GEN            = 0x0000402dL;
+
+    public static final long  CKM_SP800_108_COUNTER_KDF      = 0x000003acL;
+    public static final long  CKM_SP800_108_FEEDBACK_KDF     = 0x000003adL;
+    public static final long  CKM_SP800_108_DOUBLE_PIPELINE_KDF = 0x000003aeL;
+
```

Comment:
Same comment.

Reply:
These three are just by themselves, so unless you feel strongly about, I prefer just leave them here which matches the ordering of pkcs11t.h, i.e. right before the CKM_VENDOR_DEFINED line.

Reply:
Just thought they could be moved like CKM_ECDSA_SHA3_224 to CKM_EDDSA (which are not matching to the ordering of pkcs11t.h), so we keep the ordering in PKCS11Constants. It's fine if you'd prefer as is.

Reply:
Thanks, I have slight preference for as is.